libraries/Microsoft.Bot.Connector/Authentication/IOAuthConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    /// <summary>
+    /// Configuration for OAuth client credential authentication.
+    /// </summary>
+    public interface IOAuthConfiguration
```

Comment:
Assuming this is only used in one place - and I don't see any Mocks or other use - do we need the interface?